src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -86,9 +86,17 @@ $ k6 run script.js
 ```
 
 ```bash
-# When using the `k6:master-with-browser` Docker image, you need to add `--cap-add=SYS_ADMIN`
-# to grant further system permissions on the host for the Docker container.
-docker run --rm -i --cap-add=SYS_ADMIN grafana/k6:master-with-browser run - <script.js
+# WARNING!
+# Be aware that grafana/k6:master-with-browser launches Chrome browser setting 'no-sandbox'
+# argument. Use it only with trustworthy websites.
+#
+# As an alternative, our recomendation is to use a Docker SECCOMP profile instead, and
```

Comment:
I ❤️ the wording.

My small suggestion is that I think it could be softened a bit:

```suggestion
# If you don't feel comfortable working without the sandbox feature in chrome, an alternative is to use a Docker SECCOMP profile instead, and
```

WDYT?

Reply:
I added @inancgumus suggestion in https://github.com/grafana/k6-docs/pull/1330/commits/99fcfbb5689ec4c1cf741fd8353e9a9aacdcaf57.

In regards of your suggestion, @ankur22 , to soften a bit the message, to be honest I'm not too convinced of the change :thinking: I don't think we need to soften it and the original message I think is more concise. I would like to hear more opinions on it, if possible.

Reply:
We could just omit the "recommendation" part, such as:
```
# As an alternative, you can use a Docker SECCOMP profile instead, ...
```
Personally I think is OK to stress the fact that, from the three options that we have found available to run chrome browser in alpine (`no-sandbox` arg,`SYS_ADMIN` capability, and the usage of a SECCOMP profile), this last one is the preferred one. We've seen this situation happening in other similar tools when running chrome browser in a container, and we also did our own investigation when building the browser farm for the cloud and got to the same conclusion that using the SECCOMP is the preferred way. Happy to hear more feedback on this!

Reply:
I think the more concise version of `As an alternative, you can use a Docker SECCOMP profile instead, ...` is great.

Reply:
> I think the more concise version of As an alternative, you can use a Docker SECCOMP profile instead, ... is great.

I like this option as it's more concise and does not mention the word "recommendation". Changed in [7b98b68](https://github.com/grafana/k6-docs/pull/1330/commits/7b98b68942a04393fb1e2f587c2e1f0d570d61e3).